credsweeper/rules/config.yaml

```diff
@@ -6,6 +6,7 @@
   filter_type:
     - CreditCardNumberCheck
   min_line_len: 16
+  required_regex: ([a-zA-Z0-9_/+-]{15,})
```

Comment:
Checking the line has more than 15 characters is proved to increase the performance of `required_regex`?
I asked this, because the performance might be increased by code block of `Scanner` class.
As you can see checking regex function is changed with `search()`.
https://github.com/Samsung/CredSweeper/blob/235ccce812a69598a9216ff5bdcbf22cfbc17d83/credsweeper/scanner/scanner.py#L139-L146

Reply:
``search`` was replaced to ``match``  https://github.com/Samsung/CredSweeper/pull/388/commits/d3dd3653cd50dcfbe1ab36a7d301be2615614192
locally speedup is 2%

Reply:
I see, thank you for explanation.